science/PothosLiquidDSP/Portfile

```diff
@@ -19,37 +19,37 @@ version             0.0.0-git-20201204
 checksums           rmd160  7425ddaaa3df186de5ffb83a9b7564f9dd32145c \
                     sha256  1dc25c7a9a46bd2d80d4b8dee5179c13d25c8b3437de143249064f437ba97e74 \
                     size    37797
-revision            0
+revision            1
 
 compiler.cxx_standard 2011
 
 # set Python version and variants
 set PythonVersionNoDot ""
 
 # select Python to use; default to 3.6
-variant python35 conflicts python36 python37 description {Build using Python 3.5} {}
-variant python36 conflicts python35 python37 description {Build using Python 3.6} {}
 variant python37 conflicts python35 python36 description {Build using Python 3.7} {}
+variant python38 conflicts python35 python36 description {Build using Python 3.8} {}
```

Comment:
Need to fix up conflicts with new variants

Reply:
Oh right ;-) It was a propose. If you accept I will remove all variants since are useless (this simplify a lot the port).

Reply:
hmmm ... I'm very mixed about whether to remove `+python*` variants or not. I would prefer to keep variants around, since end-users will want to select a specific Python version & just use that, which may or not be the one we've selected. Different ports will require different Python versions, which are guaranteed to not overlap with that we've selected. By providing `+python*` variants we can keep these various ports with the same Python version. So ... I guess I've talked myself into wanting to keep some variants, just change the default to Py39. Does this make sense to you? Please provide logic if you feel otherwise!

Reply:
My considerations were:
- any gr module that works with python 3.7 it works fine with 3.9 (or at least ~99.9999%) and the performance are better (since also debian stable moved to 3.9);
- simplify a lot the ports without creating a new portgroup to manage the complexity and future changes;
- I don't think many users are selecting different version since point 1.
- many many python ports have moved to 3.9
- obviously, when we select a python version we should maintain it stables for a few years (at least)
- all gr-* modules, gnuradio and related packages will use the same python version (I already using 3.9 for everything)

At this point, I don't catch the real necessity of having multiple python versions except for performance comparisons.

Reply:
Very good arguments!